src/create-theme-provider.js

```diff
@@ -0,0 +1,92 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+const isFunction = require('is-function');
```

Comment:
Actually iframe use case would work with typeof, I mixed it up with the .constructor check which wouldn't.